Combobox/ComboPopup.js

```diff
@@ -58,6 +58,11 @@ define([
 							.removeClass("d-hidden");
 					}
 					this.combobox._prepareInput(this.inputNode);
+					this.combobox.observe(function (oldValues) {
+						if ("opened" in oldValues) {
+							this.inputNode.setAttribute("aria-expanded", this.combobox.opened);
+						}
+					}.bind(this));
```

Comment:
`aria-expanded` should indicate whether or not the list is displayed, which (on the master branch) is controlled by `Combobox#_togglePopupList()`, or you can get the info from `this.combobox.list.getAttribute("d-shown")`.

Usually the list is displayed, but sometimes it isn't, like the second to last test case in ComboPopup.html where hasDownArrow=false minFilterChars=3.  The list is only displayed after typing 3 characters.